instrumentation/ratpack-1.4/src/test/groovy/server/RatpackHttpServerTest.groovy

```diff
@@ -137,4 +135,11 @@ class RatpackHttpServerTest extends HttpServerTest<EmbeddedApp> {
   String expectedServerSpanName(String method, ServerEndpoint endpoint) {
     return endpoint.status == 404 ? "/" : endpoint == PATH_PARAM ? "/path/:id/param" : endpoint.path
   }
+
+  @Override
+  boolean testException() {
+    // TODO(anuraaga): We record exception both in the ratpack controller and in the error handler.
+    // Do we need the latter? https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/810
+    return false
```

Comment:
If this is `false`, how can the `if` on line 128 be ever `true`?

Reply:
It can't - but I would remove that if along with the TODO if this can't ever be fixed, similar to all the tests I've only supressed the exception cases for further debugging.